tests/phpunit/integration/api/fields/includes/class-fields-test-case.php

```diff
@@ -58,21 +58,6 @@ public function tearDown() {
 		parent::tearDown();
 	}
 
-	/**
-	 * Get the value of the private or protected property.
-	 *
-	 * @since 1.5.0
-	 *
-	 * @param string $property Property name for which to gain access.
-	 *
-	 * @return mixed
-	 * @throws \ReflectionException Throws an exception if property does not exist.
-	 */
-	protected function get_reflective_property_value( $property ) {
```

Comment:
I notes in the Integration Test Case that this method is not effective at the base level.  Why? Some class constructs require arguments to instantiate the object.  We have a choice here as it's only used in one spot:

1. Get rid of this method completely, moving the code back to calling test method.
2. Modify it to allow the class name to be passed in.

Let's go with option 1.